File: Assignment__11/Assignment_java/Assignment_4.java
import java.util.*;

class Pattern 
{
    public void Display(int iNo)
    {
        int iCnt = 0;
       
        for(iCnt = 1 ; iCnt <= iNo ; iCnt++)
        {
            System.out.print("#\t"+iCnt+"\t*\t");
        }
        System.out.println();
    }
}
class Assignment_4
{


    public static void main(String A[])
    {   
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number : ");
        int iValue = sobj.nextInt();

        Pattern pobj = new Pattern();

        pobj.Display(iValue);
    }
}

File: Assignment___41/Assignment_4.c
#include<stdio.h>
#include<stdlib.h>

void Display(int iNo)
{
    static int i = 1;
    static char No ='A' ;
    if(i <= iNo)
    {
        printf("%c\t",No);
        No++;
        i++;
        Display(iNo);
    }
}

int main()
{
    int iValue = 0 ;
    printf("Enter number : ");
    scanf("%d",&iValue);

    Display(iValue);
    printf("\n");

    return 0;
}